Example/Tests/DSTransactionTests.m

```diff
@@ -389,5 +389,84 @@ - (void)testResetBlockchainIdentityTransactionInputs {
 //
 //}
 
+/**
+ * Inputs:
+ * 1) hashes (in reversed byte-order) lexicographically in ASC.
+ * 2) indices  in ASC.
+ * Outputs:
+ * 1) amounts in ASC.
+ * 2) outScripts lexicographically in ASC
+ */
+- (void)testBIP69 {
+    DSChain *devnetDRA = [DSChain testnet];
+    NSString *inputPrivateKey = @"cNeRqjZpEEowdxMjiBa7S5uBgqweng19F1EZRFWcqE2XTpDy1Vzt";
+
+    DSECDSAKey *k2 = [DSECDSAKey keyWithSecret:*(UInt256 *)@"0000000000000000000000000000000000000000000000000000000000000001".hexToData.bytes compressed:YES];
+    DSECDSAKey *k3 = [DSECDSAKey keyWithSecret:*(UInt256 *)@"0000000000000000000000000000000000000000000000000000000000000002".hexToData.bytes compressed:YES];
+    DSECDSAKey *k4 = [DSECDSAKey keyWithSecret:*(UInt256 *)@"0000000000000000000000000000000000000000000000000000000000000003".hexToData.bytes compressed:YES];
+    DSECDSAKey *k5 = [DSECDSAKey keyWithSecret:*(UInt256 *)@"0000000000000000000000000000000000000000000000000000000000000004".hexToData.bytes compressed:YES];
+
+    NSString *a1 = @"yaMmAV9Fmx4St7xPH9eHCLcYJZdGYd8vD8";
+    NSString *a2 = [k2 addressForChain:devnetDRA];
+    NSString *a3 = [k3 addressForChain:devnetDRA];
+    NSString *a4 = [k4 addressForChain:devnetDRA];
+    NSString *a5 = [k5 addressForChain:devnetDRA];
+
+    NSMutableData *script1 = [NSMutableData withScriptPubKeyForAddress:a1 forChain:devnetDRA];
+    NSMutableData *script2 = [NSMutableData withScriptPubKeyForAddress:a2 forChain:devnetDRA];
+    NSMutableData *script3 = [NSMutableData withScriptPubKeyForAddress:a3 forChain:devnetDRA];
+    NSMutableData *script4 = [NSMutableData withScriptPubKeyForAddress:a4 forChain:devnetDRA];
+    NSMutableData *script5 = [NSMutableData withScriptPubKeyForAddress:a5 forChain:devnetDRA];
+
+    NSValue *hash1 = uint256_obj(*(UInt256 *)@"ce5e6919b13d6e58da10f933b6442558ba470b24f488f1449d2adf1e0a892e7d".hexToData.reverse.bytes);
```

Comment:
You can now do @"ce5e6919b13d6e58da10f933b6442558ba470b24f488f1449d2adf1e0a892e7d".hexToData.reverse.UInt256 which is a little easier. (Other unit tests that you might have copied this from should probably use this notation.)